pages/order.vue

```diff
@@ -158,6 +125,27 @@
       }
     },
     computed: {
+      planTermOptions() {
+        const monthlyPlan = {
+          title: 'Monthly',
+          value: 'month'
+        }
+        const yearlyPlan = {
+          title: `Yearly (save additional ${this.calculateSavings('year', 1)}%)`,
+          value: 'year'
+        }
+        return [monthlyPlan, yearlyPlan]
+      },
+      orderPricesTableData() {
+        return this.usersRangeKeys.map(range => ({
```

Comment:
This computed property and the table are _tightly coupled_.  That is not good.

I think the proper separation would be for it to take a plan as a prop, then have the table component generate the table data itself. It would have to get the list of range keys and the calculateSavings function from somewhere... for now those could also be props, although they may benefit from a refactor to a composition function.